pandas/tests/reshape/test_pivot.py

```diff
@@ -185,6 +185,45 @@ def test_pivot_table_dropna(self):
         tm.assert_index_equal(pv_col.columns, m)
         tm.assert_index_equal(pv_ind.index, m)
 
+    def test_pivot_table_keep_nancols(self):
+        df = pd.DataFrame(
```

Comment:
Can you add the issue number as a comment here?

Reply:
done